func_programming/8.py

```diff
@@ -0,0 +1,11 @@
+orders = [
+["34587", "Learning Python, Mark Lutz", 4, 40.95],
+["98762", "Programming Python, Mark Lutz", 5, 56.80], ["77226", "Head First Python, Paul Barry", 3,32.95],
+["88112", "Einführung in Python3, Bernd Klein", 3, 24.99]
+]
+
+calculated_orders = lambda order: (order[0], order[2] * order[3] if order[2] * order[3] >= 100 else order[2] * order[3] + 10)
```

Comment:
dobre rozwiązanie w użyciem lambd, oczywiście gdyby to nie było zadanie związane z lambdami, zapisalibyśmy to czytelniej :)